main.go

```diff
@@ -60,8 +58,6 @@ func main() {
 		logger.Fatalf("ERROR: Failed to initialise OAuth2 Proxy: %v", err)
 	}
 
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Deprecated: As of Go 1.20 there is no reason to call Seed with a random value.  https://pkg.go.dev/math/rand#Seed